programs/uxd/src/instructions/credix_lp/edit_credix_lp_depository.rs

```diff
@@ -132,6 +134,20 @@ pub(crate) fn handler(
         });
     }
 
+    // optional: rebalancing_target_weight
```

Comment:
Issue with adding it to each depository is that you must have all depositories to validate the value is correct. (It's a relative value)

As such, I recommend pulling it out of the depositories into the controller (as suggested in the Slack message), that way you can modify and make sure the total is still 100%?